sessionctx/variable/sysvar.go

```diff
@@ -745,6 +745,30 @@ var defaultSysVars = []*SysVar{
 			return nil
 		},
 	},
+	{Scope: ScopeGlobal, Name: TiDBEnablePrepPlanCache, Value: BoolToOnOff(DefTiDBEnablePrepPlanCache), Type: TypeBool, SetGlobal: func(s *SessionVars, val string) error {
+		EnablePreparedPlanCache.Store(TiDBOptOn(val))
+		return nil
+	}, GetGlobal: func(s *SessionVars) (string, error) {
+		return BoolToOnOff(EnablePreparedPlanCache.Load()), nil
+	}},
+	{Scope: ScopeGlobal, Name: TiDBPrepPlanCacheSize, Value: strconv.FormatUint(uint64(DefTiDBPrepPlanCacheSize), 10), Type: TypeUnsigned, MinValue: 1, MaxValue: 100000, SetGlobal: func(s *SessionVars, val string) error {
```

Comment:
For combining these two vars into one, I'm afraid it can cause some compatibility issues.
For example, if the original config is `Enable=False && CacheSize=500`, then both choices(`0` and `500`) of `CacheSize` are not appropriate. 
If we set it to `0`, then the original value of `CacheSize=500` will be lost, and if we set it to `500`, then the plan-cache will be enabled automatically. So keeping two sysvars here seems safer.

Reply:
OK, no problem :-)